public/js/products.js

```diff
@@ -1 +1,39 @@
-// For future use
+let counter = 4
+let rowNum = 1
+
+function getProductResults() {
+  $.getJSON('../api/products.json', function (element) {
+    element.forEach((arr) => {
+      if (counter === 4) {
+        // Create a new bootstrap row after every 4th result and in the very beginning.
+        // Row IDs created to keep track of every row to know where to insert every column (1 search result).
+        generateRow('product-row-num-' + rowNum)
+        counter = 0
+        rowNum++
+      }
+
+      //Create new product image and title. Four are occupied per row.
+      generateColumn(arr.image, arr.name, rowNum - 1)
+      counter++
+    })
+  })
+
+  function generateRow(idNum) {
+    $.get('/components/product-row.html', function (data) {
```

Comment:
find a way to call these functions once, store it in a string variable and reuse it for every item